vendor/github.com/asaskevich/govalidator/patterns.go

```diff
@@ -4,49 +4,55 @@ import "regexp"
 
 // Basic regular expressions for validating strings
 const (
```

Comment:
these updates are problematic, they change the API validation of CRD format validators

@jpbetz @sttts @deads2k  we should isolate use of these in kube-openapi and stop depending on an external library for this API 
surface